src/frontend-admin/src/components/Group/Group.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import styles from './Group.module.css'
+import cx from 'classnames' //allows use of more than 1 classname
+
+//Types
+type JustifyType = 'start' | 'end' | 'between' | 'around' | 'even'
+type DirectionType = 'row' | 'column'
+
+interface IProps {
+    children: JSX.Element[], 
```

Comment:
I somewhat regret using this type in my old repo (I'm assuming that's what I did). This is a rare case where `any` I think is the appropriate typing. The children can be lots of different things: elements, strings, conditional elements (like `{isModalVisible && <MyModal />}` and this particular typing is too restrictive to accommodate those